webview/src/plots/components/ribbon/styles.module.scss

```diff
@@ -10,7 +10,7 @@
 .block {
   border-left-style: solid;
   border-left-width: 3px;
-  background-color: $header-bg-color;
+  background-color: var(--vscode-dropdown-background);
```

Comment:
We try to keep all colors inside the `variables.scss` file so that it's easy to see what's available and what we are using (so that we don't start having too many different colors). The fact that the new `header-bg-color` impacts plots as well as experiments might not be a big problem if the color fits.

Reply:
> We try to keep all colors inside the variables.scss file so that it's easy to see what's available and what we are using (so that we don't start having too many different colors). 

Oh, makes sense! Thanks for letting me know!

> The fact that the new header-bg-color impacts plots as well as experiments might not be a big problem if the color fits.

Color did not fit (made them have no background at all) 😄